src/components/Header.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useState } from "react";
+import { Transition } from "@headlessui/react";
+import Image from "next/image";
+
+export default function Header({
+  index,
```

Comment:
Why does this take in index and when is it used?

Reply:
removed type script type check